samples/app-ReporterPlus/ReporterPlus/Helpers/CardHelper.cs

```diff
@@ -0,0 +1,99 @@
+﻿using AdaptiveCards.Templating;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+using System.IO;
+using ReporterPlus.Models;
+
+namespace ReporterPlus.Helpers
+{
+    public class CardHelper
+    {
+        public static Attachment CreateAdaptiveCardAttachment(string cardType, BlobDataDeserializer blobData, string channel, out string cardJsonstring)
+        {
+            Attachment cardAttachment;
+            switch (cardType)
+            {
+                case "BaseCard":
```

Comment:
Please check out my suggestion the ReporterPlusBot.cs and try to update these cases to use the new class or enum. Same in line number 22, 31, 40 etc.